doc/userguide/src/Appendices/DocumentationFormatting.rst

```diff
@@ -110,22 +54,21 @@ the same two line documentation.
 __ `Dividing test data to several rows`_
 __ Escaping_
 
-.. table:: Multiline documentation examples
-   :class: example
-
-   =========  ===============  ================  ==============  ==============
-   Test Case      Action           Argument         Argument       Argument
-   =========  ===============  ================  ==============  ==============
-   Example 1  [Documentation]  First line\\n     Second line in  multiple parts
-   \          No Operation
-   Example 2  [Documentation]  First line
-   \          ...              Second line in    multiple parts
-   \          No Operation
-   Example 3  [Documentation]  First line\\n
-   \          ...              Second line in\\
-   \          ...              multiple parts
-   \          No Operation
-   =========  ===============  ================  ==============  ==============
+.. sourcecode:: robotframework
+
+  *** Test Cases ***
+   Example 1  
+       [Documentation]    First line\n    Second line in    multiple parts
+       No Operation
+   Example 2
```

Comment:
Should have empty lines between tests. Apparently the original hadn't either, but I think all examples should have them.